http/test/test_client.lua

```diff
@@ -27,6 +27,47 @@ if not(resp.code == 200) then error("resp code") end
 if not(resp.body == "OK") then error("resp body") end
 print("done: http.client:getBasicAuth")
 
+local req, err = http.file_request("http://127.0.0.1:1111/upload", {
+  fieldname = "file",
+  path = "./test/data/test.txt"
+})
+if err then error(err) end
+local resp, err = client_1:do_request(req)
+if err then error(err) end
+if not(resp.code == 200) then error("resp code") end
```

Comment:
**suggestion; not-blocking**: Similarly this could be the following which would also include the value observed in the error message

```suggestion
assert(resp.code == 200, "resp code: "..resp.code)
```